### DISCUSSION TITLE
Is there roles + permissions concept?

### DISCUSSION THREAD

**Discussion by ISulimanov** [zitadel/zitadel#5298] [Q&A] [answered]
What a right way to handle permissions and roles concept in Zitadel? 

Let's say a have follow permissions: 
 - Create project 
 - Edit Issue

and I would like create a role - Project Manager that will include all above permissions.

**Comment by hifabienne** [comment, marked as answer]
Hei @ISulimanov 

At the moment ZITADEL does only handle the roles and not the permissions.
So you have 2 possibilities:
Possibility 1:
1. Create a role project manager
2. Your application has a mapping that the role project manager has the permissions create project and edit issue.
3. When the user accesses your application with the role project manager, the app knows that the user is allowed to  create projects and edit issues.

Possibility 2:
1. You create the role create project and edit issue in ZITADEL. You can give them the group Project Manager. Be aware that the group has is only an attribute and has no impact. E.g. ZITADEL console uses it so you can select all roles of a group, to give access to users faster.
2. The app knows the permissions create project and edit issue.
3. When the user accesses the app with the roles, the app knows what to do

**Reply by ISulimanov** [reply]
@hifabienne thank you, Do you have any plans to implement permissions in future?

**Reply by hifabienne** [reply]
So far we do not have the plan to implement this.
If we see a high demand for this topic rising, we will probably discuss it.
